cypher/cypher-docs/src/docs/dev/ql/administration/security/grant-deny-syntax.asciidoc

```diff
@@ -39,23 +39,23 @@ Graphs created after this command execution will also be associated with these p
 | Command | Description
 
```

Comment:
Further up in this file we link to read and write privileges, in their list of privileges is `{props}`, those should have been updated as well

Reply:
there's also a `ELEMENTS+*+` which can be updated at the same time (shouldn't have `+`s)

Reply:
(if the `+` are removed confirm that the text doesn't get bold)

Reply:
The `+` might have been fixed already but still looks slightly odd with a non-escaped space in between: `ELEMENTS` `*`

Reply:
See, https://github.com/neo4j/neo4j-documentation/pull/762#discussion_r528561758